dynamiqs/options.py

```diff
@@ -44,6 +44,7 @@ def __init__(
         verbose: bool = True,
         dtype: jnp.complex64 | jnp.complex128 | None = None,
         cartesian_batching: bool = True,
+        progress_bar: bool = True,
```

Comment:
This is redundant with `verbose`. `progress_bar` is more explicit but longer, pick what you prefer!